src/USDA.NET/BaseClient.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+
+namespace USDA.NET
+{
+    public class BaseClient : IDisposable
```

Comment:
`abstract` should be added here since there are no public constructors.

Reply:
I initially created this class thinking that I would create a separate "client' for the food report, but I think all of the endpoints could be contained in FoodCompositionClient. BaseClient could be rolled back into FoodCompositionClient